 builds/unix/unix-cc.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/builds/unix/unix-cc.in b/builds/unix/unix-cc.in
index 30d097b..4b4085d 100644
--- a/builds/unix/unix-cc.in
+++ b/builds/unix/unix-cc.in
@@ -122,7 +122,6 @@ CCexe        := $(CCraw_build)	# used to compile `apinames' only
 #
 LINK_LIBRARY = $(LIBTOOL) --mode=link $(CCraw) -o $@ $(OBJECTS_LIST) \
                           -rpath $(libdir) -version-info $(version_info) \
-                          $(LDFLAGS) -no-undefined \
-                          -export-symbols $(EXPORTS_LIST)
+                          $(LDFLAGS) -no-undefined
 
 # EOF
