 builds/freetype.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builds/freetype.mk b/builds/freetype.mk
index 1d7eeb6..9370226 100644
--- a/builds/freetype.mk
+++ b/builds/freetype.mk
@@ -159,7 +159,7 @@ FT_COMPILE := $(CC) $(ANSIFLAGS) $(INCLUDE_FLAGS) $(FT_CFLAGS)
 
 # Include the `exports' rules file.
 #
-include $(TOP_DIR)/builds/exports.mk
+#include $(TOP_DIR)/builds/exports.mk
 
 
 # Initialize the list of objects.
